src/styles/shared/_breakpoints.scss

```diff
@@ -191,3 +193,15 @@ $nav-min-window: em(layout-width(page-with-nav));
     @content;
   }
 }
+
+@mixin page-before-resource-list-small() {
+  @include breakpoint-before($resource-list-small) {
+    @content;
+  }
+}
+
+@mixin page-after-resource-list-small() {
+  @include breakpoint-after($resource-list-small) {
+    @content;
+  }
+}
```

Comment:
Adding mixins for resource list breakpoints to remove reliance on the scss file in `components/ResourceList`

Reply:
This file is part of the [public sass API](https://github.com/Shopify/polaris-react/blob/master/src/styles/_public-api.scss). I'm guessing we don't want to expose these helpers to the world.

We've got the concept of the private api - mixins that we can use internally but aren't exposed to the world in https://github.com/Shopify/polaris-react/blob/master/src/styles/_common.scss.

In order to not grow the public api (which is traditionally basically impossible things to remove things from) can we put these into a new file that is only imported from _common.css so it is available internally but not externally.

Reply:
@BPScott I took a look at the files and saw they were almost the same. Didn't realize we were just creating new files and not adding them to the public file. I can do that now 🙌